code-challenges.test.js

```diff
@@ -1,69 +1,316 @@
+
+
+//----- I created a function called capFirst
 
+const capFirst = () => {
 
-// b) Create the function that makes the test pass.
+//----- I used the .map() to locate and capitalize .toUpperCase the index of the first letter of each word in randomNouns1 and randomNouns2
+    
+  let firstCap = randomNouns2.map(value => {
 
+    return value.toUpperCase() + value.substring(1)
+    })
 
+//----- I used the .join(" ") to get the array of strings to display as expected    
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+    return firstCap.join(" ")
+}
 
-// a) Create a test with expect statements for each of the variables provided.
+//----- console.log results
 
-const vowelTester1 = "learn"
+console.log(capFirst())
+
+//---- My Expected Output : "Temperature", "Database", "Chopsticks", "Mango", "Deduction"
+//---- My Output : "Temperature", "Database", "Chopsticks", "Mango", "Deduction"
+
+
+
+// const capFirst = () => {
+
+//    let firstCap = randomNouns1.map(value => {
+    
+//     return value.toUpperCase() + value.substring(1) 
+// })
+//      return firstCap.join(" ")
+// }
+// console.log(capFirst())
+
+//---- My Expected Output : "Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"
+//---- My Output : "Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"
+
+
+
+//3//
+// // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+ const vowelTester1 = "learn"
 // Expected output: 1
-const vowelTester2 = "academy"
+ const vowelTester2 = "academy"
 // Expected output: 0
-const vowelTester3 = "challenges"
+ const vowelTester3 = "challenges"
 // Expected output: 2
 
 
-// b) Create the function that makes the test pass.
+//------- MY TEST-----------
+
+describe("findFirstVowel", () => { 
+
+ it("use .split to split vowelTester1, then use indexOf to index the first instance of a vowel", () => {
+  
+   expect(findFirstVowel(vowelTester1)).toEqual(1)
+  
+  })
+})
+
+describe("findFirstVowel", () => { 
+
+    it("use .split to split vowelTester2, then use indexOf to index the first instance of a vowel", () => {
+     
+      expect(findFirstVowel(vowelTester2)).toEqual(0)
+     
+     })
+   })
+
+describe("findFirstVowel", () => { 
+
+    it("use .split to split vowelTester3, then use indexOf to index the first instance of a vowel", () => {
+     
+      expect(findFirstVowel(vowelTester3)).toEqual(2)
+     
+     })
+   })
+
+
+// // b) Create the function that makes the test pass.
+
+
+//---- I created a function called findFirstVowel
+
+const findFirstVowel = () => {
+
+//----- I used .split("") to separate the letters in the string
+
+const firstVowel = vowelTester1.split("")
+
+//----- I used .indexOf to reference the first instance of a vowel
+
+    return firstVowel.indexOf("a", "e", "i", "o", "u")  
```

Comment:
I really like your thought process on this approach.  Unfortunately, Javascript is super finicky and only allows one letter to be passed as an argument in the indexOf method.  This actually works for the second two instances as the first vowel on both are "a" which is the first (and only) argument, but this doesn't take into account any other vowel.